Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/TeleopDrive.java

```diff
@@ -0,0 +1,62 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package org.usfirst.frc.team199.robot.commands;
+
+import org.usfirst.frc.team199.robot.Robot;
+import org.usfirst.frc.team199.robot.motion.PID;
+
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ * An example command.  You can replace me with your own command.
+ */
+public class TeleopDrive extends Command {
+	public TeleopDrive() {
+		// Use requires() here to declare subsystem dependencies
+		requires(Robot.dt);
+	}
+
+	// Called just before this Command runs the first time
+	@Override
+	protected void initialize() {
+	}
+
+	// Called repeatedly when this Command is scheduled to run
+	@Override
+	protected void execute() {
+		if(SmartDashboard.getBoolean("Arcade Drive", true)) {
+			if(SmartDashboard.getBoolean("Arcade Drive Default Setup", true)) {
+				Robot.dt.arcadeDrive(Robot.oi.leftJoy.getY(), Robot.oi.rightJoy.getX());
```

Comment:
can you make these drive functions methods in the Drivetrain subsystem so they're not only here in the command? Bc it is a function of the subsystem, even though it's only really used here.

Reply:
Done, see https://github.com/DeepBlueRobotics/RobotCode2018/commit/dbf634b2bccc3cfa9950e91d26077f166722134c